llvm/lib/Target/AMDGPU/AMDGPU.h

```diff
@@ -369,6 +369,17 @@ extern char &AMDGPUCodeGenPrepareID;
 void initializeAMDGPURemoveIncompatibleFunctionsPass(PassRegistry &);
 extern char &AMDGPURemoveIncompatibleFunctionsID;
 
+void initializeAMDGPUDemoteSCCBranchToExeczLegacyPass(PassRegistry &);
+extern char &AMDGPUDemoteSCCBranchToExeczLegacyID;
+
+class AMDGPUDemoteSCCBranchToExeczPass
```

Comment:
Move this to a header AMDGPUDemoteSCCBranchToExecz.h and include it in AMDGPUDemoteSCCBranchToExecz.cpp.